app/websocket/src/main/kotlin/com/xorker/draw/websocket/session/SessionManager.kt

```diff
@@ -34,4 +37,12 @@ internal class SessionManager {
         val session = sessionMap.remove(sessionId)
         return userIdMap.remove(session?.user?.id)
     }
+
+    fun setPing(sessionId: SessionId) {
+        val session = sessionMap[sessionId]
+
+        session?.let {
+            session.ping = true
+        }
+    }
```

Comment:
p3. let 으로 null check 하는 것은 권장하지 않습니다.
https://tourspace.tistory.com/208

```suggestion
        val session = sessionMap[sessionId] ?: return
        session.ping = true
    }
```